workspace/notebook/py_sb_horizon_harmonised_nsip_exam_timetable.json

```diff
@@ -20,7 +20,7 @@
 				"spark.dynamicAllocation.enabled": "false",
```

Comment:
Just a thought, since this mapping is required by the applications team, should we not have done the mapping directly in `nsip_exam_timetable_migration` notebook which builds the migration table rather than applying it in the harmonised layer which might effect appeals or other consumers?